roles/network/tasks/main.yml

```diff
@@ -102,8 +103,6 @@
     - name: Restart services (and clone wifi to create ap0 if necessary)
       include_tasks: restart.yml
 
-  # end block
```

Comment:
Can we keep the original Line 105, at minimum?

(The "# end block" visual indicator can be priceless!  Moving the `when:` clause to the top of a block is counterintuitive, but if it's tested and Ansible strongly recommends this new way of doing things, ok we can try it!)